File: examples/scripts/custom_script_priority6.py
#!/usr/local/bin/python
import time
from std_msgs.msg import Bool
import signal
import sys
import rospkg

rospack = rospkg.RosPack()
mission_control_path = rospack.get_path('mission_control')

sys.path.append("%s/src" % mission_control_path)

import mission_control_utils

mission_control_utils.ros_init("custom_script_priority6")

counter6 = 10
execute = False

for i in range(counter6, 0, -1):
    counter6 -= 1
    mission_control_utils.set_var("counter6", counter6)

    print "Countdown priority 6: %d" % counter6

    time.sleep(1)

File: examples/scripts/custom_script_priority6.cpp
#include <iostream>
#include <stdio.h>
#include <string>
#include <mission_control/mission_control_utils.h>

int main (int argc, char** argv)
{
  ros_init("mission_control_utils");

  std::string name = "counter6";
  std::string def_val = "10";

  for(int i = 0; i < 10; i++)
  {

    printf("In cpp file counter6 %d\n", i);
    set_var(name, std::to_string(i), 5);
    sleep(1);

    std::string var = get_var(name, def_val);
    printf("In cpp got counter6 %s\n", var.c_str());
  }

  for(int i = 0; i < 10; i++)
  {
    printf("In cpp file counter6 %d\n", i);
    sleep(1);
    std::string var = get_var(name, def_val);
    printf("In cpp got counter6 %s\n", var.c_str());
  }

  return 0;
}
